go/common/host/services.go

```diff
@@ -111,8 +95,6 @@ type L1Publisher interface {
 	// PublishSecretResponse will create and publish a secret response tx to the management contract - fire and forget we don't wait for receipt
 	PublishSecretResponse(secretResponse *common.ProducedSecretResponse) error
```

Comment:
I don't quite get it.
Isn't "Service" supposed to be a generic interface implemented by all Services that a host depends on?
Or is this approach different?